### ISSUE TITLE
sklearn and skimage not included in dependencies

### ISSUE THREAD

**Issue by nic-kill** [Astroua/TurbuStat#231]
Not sure if this is only on my end as i've had some issues with modules recently which is what precipitated this discovery but for me on a fresh install of turbustat the import lines:

`from turbustat.statistics.apodizing_kernels import (CosineBellWindow, TukeyWindow, HanningWindow, SplitCosineBellWindow)`

`from turbustat.statistics import PowerSpectrum`

both return 

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/nickill/software/python/turbustat/statistics/__init__.py", line 6, in <module>
    from .pca import *
  File "/home/nickill/software/python/turbustat/statistics/pca/__init__.py", line 1, in <module>
    from .pca import PCA, PCA_Distance
  File "/home/nickill/software/python/turbustat/statistics/pca/pca.py", line 13, in <module>
    from .width_estimate import WidthEstimate1D, WidthEstimate2D
  File "/home/nickill/software/python/turbustat/statistics/pca/width_estimate.py", line 13, in <module>
    from skimage.measure import find_contours
ModuleNotFoundError: No module named 'skimage'
```

with a similar message saying no module sklearn. This may be related to the naming as from a quick googling it seems they are also referenced as scikit-learn and scikit-image. for me a manual install of these two packages fixed the issue, specifically i had to do:
`pip install scikit-image`
rather than:
`pip install skimage`

this is for turbustat v1.1

**Comment by e-koch** [comment]
@nic-kill, thanks for reporting the problem!

These install issues are fixed in the development version which I haven't released on pip. Installing the missing packages via pip works great, but in case anyone else encounters this issue, you can also build the development version via pip using:
```
pip install git+https://github.com/Astroua/TurbuStat.git
```

**Comment by e-koch** [comment]
I've released v1.2 in pip, so this issue should be fixed now.
